lib/gitsh/input_strategies/interactive.rb

```diff
@@ -13,6 +13,7 @@ module Gitsh
   module InputStrategies
     class Interactive
       BLANK_LINE_REGEX = /^\s*$/
+      CONTINUATION_PROMPT = '> '.freeze
```

Comment:
We don't use `PS1` for the main prompt (there's a git-config(1) variable called `gitsh.prompt` instead). If we were going to support customisation, I think we should follow that pattern instead.